README.md

```diff
@@ -39,23 +45,34 @@ before_script:
 ```yaml
 matrix:
   include:
-    - rvm: 2.3.1
+    - rvm: 2.3.3
       script:
         - bundle exec danger
 ```
 
-And add the `DANGER_GITHUB_API_TOKEN` as an environment variable in
-Travis's repo settings, which are at a URL like
-`https://travis-ci.org/ministryofjustice/[repo]/settings`. The API token
-itself is in Rattic (search for `Danger`).
+Commit and push the changes.
+
+1. Add the MoJ Dangerbot API token to Travis;
+
+This assumes your project has already been added to Travis.
+
+    https://travis-ci.org/ministryofjustice/[your project]/settings
+
+Add the environment variable `DANGER_GITHUB_API_TOKEN`, with the value
```

Comment:
Is it worth being specific (or linking to documentation) about adding environment variables? Might be overkill, though.